test-configs.yaml

```diff
@@ -62,19 +62,27 @@ test_plan_default_filters:
         - ['i386', 'i386_defconfig']
         - ['x86_64', 'x86_64_defconfig']
 
+  - blacklist: &clang_environment_filter
+      build_environment: ['clang']
+
+  - blacklist: &kselftest_defconfig_filter
+      defconfig: ['kselftest']
+
+  - blacklist: &kselftest_defconfig_clang_environment_filter
```

Comment:
We can also just have the 2 aliases above listed in each test plan.  I'm not sure if it's that helpful to have clang kselftest filters tied together into one alias.

Reply:
Yeah I wasn't sure either, but having just learned about YAML anchoring I liked the less duplication

Reply:
Fair enough, it's easy to adjust again in the future to whatever appears most practical.